### PR TITLE
Adjusted SP formula for rho

### PR DESCRIPTION
**Pull request by summatix**
image

### CODE REVIEW

**Review by lokdao — COMMENTED** [review]
Any reason why the explanation of the intermediate variable is not there?
I recall in the PR under the rewards calculation tool repo we had more explanations (cf: https://github.com/IndigoProtocol/rewards/pull/14)

Similar to my comment here https://github.com/IndigoProtocol/paper/pull/5, any reason for using this notation `|σ|` instead of `n` or `N` ?

**Review by lokdao — APPROVED** [review]

**Comment by advaitrane98** [comment]
@summatix all the summations start from i=0, so it should go till |n|-1?

**Comment by lokdao** [comment]
@advaitrane98 , I think we could change i to start at 1.
I can see the following in section 5.2, the formula displayed seems to be inconsistent with the text.
image

**Comment by summatix** [comment]
> @advaitrane98 , I think we could change i to start at 1. I can see the following in section 5.2, the formula displayed seems to be inconsistent with the text. image

That's another error in the paper. We can open another PR to fix.

**Comment by summatix** [comment]
> @summatix all the summations start from i=0, so it should go till |n|-1?

Summations should start from 1. Will fix this PR to adjust.

**Comment by summatix** [comment]
> Any reason why the explanation of the intermediate variables is not there?
> I recall in the PR under the rewards calculation tool repo we had more explanations (cf: [IndigoProtocol/rewards#14](https://github.com/IndigoProtocol/rewards/pull/14))

The intermediate variables are simply placeholders, not external references. They don't need explanations because they could be anything and hold no meaning.

**Comment by summatix** [comment]
> Similar to my comment here #5, any reason for using this notation `|σ|` instead of `n` or `N` ?

`|σ|` is more precise and involves less English explanation, thus less prone to error

**Comment by summatix** [comment]
Created #7 to address unrelated typo

File: main.tex [hunk 1]
```diff
@@ -1259,13 +1259,13 @@ \subsubsection{Stability Pool Staking
   \(\varphi\) is the sequence of Stability Pool saturation values of
   each iAsset
 \item
-  \(\omega\) is the sequence of market caps of each iAsset
+  \(\omega\) is the sequence of market cap values of each iAsset
 \item
   \(t\) is the sequence of total number of Stability Pool stakers, who
-  have staked for at least 24 hours, of each iAsset
+  have staked for at least 24 hours, for all iAssets
```
Comment (summatix) [inline main.tex:1265]: "of each iAsset" is more consistent and I believe fits the context. "for all iAssets" changes the meaning somewhat and makes it a bit more ambiguous imo

**Review by advaitrane98 — APPROVED** [review]

**Pull request merged**